test/runtests.jl

```diff
@@ -102,6 +102,12 @@ facts("Tests for Taylor1 expansions") do
     @fact asin(t) + acos(t) == pi/2 --> true
     @fact derivative(acos(t)) == - 1/sqrt(1-t^2) --> true
 
+    @fact - sinh(t) + cosh(t) == exp(-t) --> true
+    @fact  sinh(t) + cosh(t) == exp(t) --> true
+    @fact evaluate(- sinh(t)^2 + cosh(t)^2 , rand()) == 1 --> true
```

Comment:
I understand that numerical errors propagate, and make that the actual identity is not identically fulfilled, which motivates the evaluation. Checking this in more detail, it seems that only beyond order 8, these problems appear. For the time being, we can leave this as it is.